db/migrate/20251013110710_backfill_training_periods_schedule_ids.rb

```diff
@@ -0,0 +1,17 @@
+class BackfillTrainingPeriodsScheduleIds < ActiveRecord::Migration[8.0]
+  def up
+    TrainingPeriod.find_each do |training_period|
+      contract_period = training_period.contract_period || training_period.expression_of_interest_contract_period
+
+      default_schedule = Schedule.find_by(contract_period:, identifier: "ecf-standard-september")
+
+      if default_schedule
```

Comment:
Would we not want to error out if the default schedule is not found to avoid this failing silently?